### ISSUE TITLE
This Project is DEAD

### ISSUE THREAD

**Issue by StandardLake** [perezdidac/WhatsAPINet#337]
As wonderful as this project is, it is pronounced dead. At the moment, anytime you try to use this API, it gets blocked from WhatsApp. Mr Perezdidac is not interested in updating the .Net version of (WhatsAPI) and he is not ready to welcome other people to join him in the development. So, no matter the investment in this project that you have done, you have to accept the fact that you are on your own. Rest In Peace (WhatsAPINet)

**Comment by mgp25** [comment]
@StandardLake you can use alternatives as WhatsAPI Official https://github.com/mgp25/WhatsAPI-Official

**Comment by CharlesOkwuagwu** [comment]
@StandardLake 

What prevents your from making a clone of WhatsAPINet and continuing the development yourself?

Both the PHP WhatsAPI-Official and the Python Yousup projects could serve as code check for your continued implementation.

regards.

**Comment by w00000w5** [comment]
Please mr.mgp25
Keep whatsapp api .net updated

Thanks

**Comment by mgp25** [comment]
@w00000w5 nope, dont expect that. Better you could update it yourself by comparing with the PHP source code.

Regards

**Comment by StandardLake** [comment]
Charles, can you give more insight into what you mean by Cloning and continuing the development ?

**Comment by StandardLake** [comment]
Some of us are clearly not into php and its ugly look

**Comment by abashrawi** [comment]
I created a fork here: https://github.com/abashrawi/WhatsAPINet/tree/master/WhatsAppApi 
Perhaps we can work together on it, I have applied some changes and it is working now.

**Comment by mgp25** [comment]
@abashrawi push commits here: https://github.com/mgp25/WhatsAPINet

Its an active repo i have, you can also code with reference in updates API like: https://github.com/mgp25/WhatsAPI-Official

Regards

**Comment by abashrawi** [comment]
@mgp25 Cool, I thought work on the .Net fork was dead, I'll make my commits in your active repo then.
